lbrynet/daemon/DaemonCLI.py

```diff
@@ -38,8 +38,17 @@ def set_flag_vals(flag_names, parsed_args):
 
 
 def main():
-    if len(sys.argv[1:]):
-        method, args = sys.argv[1], sys.argv[2:]
+    argv = sys.argv[1:]
+
+    # check if a config file has been specified. If so, shift
+    # all the arguments so that the parsing can continue without
+    # noticing
+    if sys.argv[1] == "--conf":
```

Comment:
this `if` should be within `if len(argv):` below  since we only check for a `--conf` argument if there are arguments.

Reply:
ops, right. will take care of that